tests/ui/src/pages/base-page.ts

```diff
@@ -0,0 +1,17 @@
+import { Locator, Page } from "@playwright/test";
+export default class BasePage {
+  constructor(readonly page: Page) {
+    this.page = page;
+  }
+
+  static readonly userId_txt = "User ID";
+  static readonly password_txt = "password";
+  static readonly continue_btn = "Continue";
+
+  //methods
+  async login() {
+    await this.page.getByLabel(BasePage.userId_txt).fill("username");
+    await this.page.getByLabel(BasePage.password_txt).fill("password");
```

Comment:
Probably a wider test/dev question related to https://nhsd-jira.digital.nhs.uk/browse/DR-1116 where the data-testid is being used for identifying elements which we want to review but if we are happy to search by label etc rather than id. 

Would be good to clarify what QA needs to identify for finding elements and if getting by label etc is sufficient

Reply:
In my view we don't need specific data-testid to identify the web elements. 
All we need is unique / distinct class name or id's or link name or labels or button names
 If we have them we don't need data-testids. 
 We can have a call to discuss on DR-1116

Reply:
I would always go for id (or data-testid either works) where possible, but agree that it needs a wider discussion

Reply:
Meeting is setup to discuss on 18th OCT